modules/ROOT/examples/installation/post-installation-steps.sh

```diff
@@ -1,11 +1,25 @@
 #!/bin/bash
 
-ocpath='/var/www/owncloud'
-ocdata='/var/www/owncloud/data'
-ocapps_external='/var/www/owncloud/apps-external'
-oldocpath='/var/www/owncloud.old'
-linkdata="/var/mylinks/data"
-linkapps-external="/var/mylinks/apps-external"
+# To setup this script for your environment, adopt the following variables to your needs:
+#
+# ocname	the name of your directory containing the owncloud files
+# ocroot	the path to ocname, usually /var/www
+# linkroot	the path to your source directory for linking data and apps-external
+# htuser	the webserver user
+# htgroup	the webserver group
+# rootuser	the root user
+
+ocname='owncloud'
+ocroot='/var/www/'
+ocpath=$ocroot$ocname
```

Comment:
How about quoting this and the following variables as an extra sanity check? Also, while it's a little more work, okay one character per variable, how about changing `ocroot` to `'/var/www'`, and then initialising the others with a forward slash between the variables, e.g., `ocpath=$ocroot/$ocname`. It could just be me, but I find compound variables easier to read when declared that way. Plus, I can't say for certain, but I thought it was an informal convention in shell scripts.